docs/pages/components/dropdowns.vue

```diff
@@ -31,6 +31,33 @@ In `KtDropdownMenus` slot, you can write your own stylied element. However, we s
 </KtDropdown>
 ```
 
+## Dropdown button 
+
+A dropdown button is a current pattern nowadays. 
+It allows to have a subaction without changing page or opening a modal.
+
+Note the `dropdownIcon` prop. It will default to `chevron_down`.
+You can also use any attribute you would pass to `KtPopover` or `KtButton`. They will be passed.
+
+<div class="element-example">
+	<KtDropdownButton size="sm" type="primary" v-t="'Dropdown button'" >
+		<div slot="KtDropdownMenus">
+			<KtDropdownMenu>Request item</KtDropdownMenu>
+			<KtDropdownMenu>Download Item</KtDropdownMenu>
+		</div>
+	</KtDropdownButton>
+</div>
+
+``` html
+<div class="element-example">
+	<KtDropdownButton size="sm" type="primary" v-t="'Dropdown button'" >
+		<div slot="KtDropdownMenus">
+			<KtDropdownMenu>Request item</KtDropdownMenu>
```

Comment:
Looks more like a `KtDropdownItem` or `KtDropdownMenuItem` to me.

Reply:
That is using the existing naming for KtDropdown :/